packages/jazz-tools/src/tools/coValues/coMap.ts

```diff
@@ -324,24 +179,24 @@ export class CoMap extends CoValueBase implements CoValue {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   toJSON(_key?: string, processedValues?: ID<CoValue>[]): any {
     const result = {
-      id: this.id,
+      id: this.$jazz.id,
       _type: this._type,
```

Comment:
@aeplay @gdorsi should I go ahead and remove `id` and `_type` from `toJSON`? That'd be consistent with viewing CoMaps as plain JSON objects

Reply:
It makes sense to me, yes.